recipe/meta.yaml

```
@@ -1,32 +1,39 @@
-{% set version = "3.06.1" %}
+{% set version = "3.08" %}
 
 package:
   name: nsis
   version: {{ version }}
 
 source:
-  - url: http://sourceforge.net/projects/nsis/files/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}.zip
-    sha256: d463ad11aa191ab5ae64edb3a439a4a4a7a3e277fcb138254317254f7111fba7
+  - url: https://altushost-swe.dl.sourceforge.net/project/nsis/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}.zip
```

```suggestion
  - url: https://downloads.sourceforge.net/project/nsis/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}.zip
```

This should work too.